api/models/Campaign.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * Models a DS Campaign.
+ */
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+
+  _id: { type: Number, index: true },
+  keyword: String,
+
+  // Properties cached from DS API.
+  title: String,
+  rb_confirmed: String,
+  rb_noun: String,
+  rb_verb: String,
+  status: String,
```

Comment:
Would it make sense to add a [`enum`](http://mongoosejs.com/docs/api.html#schema_string_SchemaString-enum) validator? Not sure if that's the case here, but if it's restricted to certain values, it would make sense.